src/theory/fp/theory_fp.cpp

```diff
@@ -309,31 +309,37 @@ Node TheoryFp::toRealUF(Node node) {
   return nm->mkNode(kind::APPLY_UF, fun, node[0]);
 }
 
-Node TheoryFp::expandDefinition(LogicRequest &lr, Node node) {
-  Trace("fp-expandDefinition")
-      << "TheoryFp::expandDefinition(): " << node << std::endl;
-
+void TheoryFp::enableUF(LogicRequest &lr) {
   if (!this->d_expansionRequested) {
     lr.widenLogic(
         THEORY_UF);  // Needed for conversions to/from real and min/max
-    lr.widenLogic(THEORY_BV);
+    //lr.widenLogic(THEORY_BV); // This has to be done when the logic is set
```

Comment:
Remove?

Reply:
Reworked so it is a comment making clear why THEORY_BV is not set in the obvious place (i.e. here).